node-webservice/react-client/src/Components/GameplayPage.jsx

```diff
@@ -4,11 +4,13 @@ import ErrorPage from './ErrorPage';
 import TestGamePage from './TestGamePage';
 import socket from '../API/socketHandler';
 import Grid from './Grid';
-
 class GameplayPage extends Component {
+  
   constructor(props) {
+   
     super(props);
     this.state = {
+      currentType : "spike",
```

Comment:
I noticed that when starting the builder page, clicking a grid cell by default adds a spike, but the type chooser by default has block selected